src/main/java/nextstep/subway/line/domain/SurCharge.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.common.exception.InvalidDataException;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class SurCharge {
```

Comment:
```suggestion
public class Surcharge {
```

어원이 sur + charge에서 왔다 하더라도 surcharge 란 단어가 존재하기 때문에 c는 소문자로 사용하는게 더 적절합니다.

Reply:
넵 수정했습니다. 피드백 감사합니다 ㅎㅎ